system76/gaze18/default.nix

```diff
@@ -0,0 +1,37 @@
+{ config, lib, pkgs, ... }:
+{
+  imports = [
+    ../.
+  ];
+  boot.initrd.kernelModules = [ "nvidia" ];
+
+  hardware.opengl = {
+    enable = lib.mkDefault true;
+    driSupport = lib.mkDefault true;
+    driSupport32Bit = lib.mkDefault true;
```

Comment:
Enabled by default:

```suggestion
```

Reply:
I'm confused, should I remove driSupport32Bit or leave it?

Reply:
I personally probably wouldn't add `driSupport32Bit` these days by default because it's only used for older 32-bit games, but I leave it to you if you want to keep it.

Reply:
I don't t think that there are any downsides from having it enabled.

Reply:
It adds something like 100M of drivers including dependencies.